orb.tmpl.yml

```
@@ -19,8 +19,13 @@ jobs:
           Additional arguments to be passed to "docker build", e.g. "--target my-stage --build-arg FOO=42".
         type: "string"
         default: ""
+      resource_class:
```

```suggestion
      resource-class:
```

for consistency with the other args; it's kinda weird but IIRC it's the prevailing convention for orb params

---

Yeah, I figured it'd be better to keep it the same as the actual param "less surprising" but i don't really care.